pom.xml

```diff
@@ -747,6 +766,22 @@
                 <artifactId>reactive-streams-tck-flow</artifactId>
                 <version>${version.lib.reactivestreams}</version>
             </dependency>
+            <dependency>
```

Comment:
These dependency management for 4th party dependencies, they belong in helidon-dependencies

Reply:
These are only internal dependencies (`reactive-streams-tck-flow`, `wagon-provider-api`, `hamcrest-library`, `weld-se-core`)
I will move the others to `dependencies`

Reply:
just to make sure we are on the same page: anything that manages the version of a dependency that may end-up under target/libs for an helidon app needs to be defined in helidon-dependencies.